tests/test_cli.py

```diff
@@ -517,3 +518,11 @@ def test_run_non_universal_newline(self):
             '\n' % path)
         self.assertEqual(
             (ctx.returncode, ctx.stdout, ctx.stderr), (1, expected_out, ''))
+
+    def test_json_output(self):
+        path = os.path.join(self.wd, 'a.yaml')
+
+        with RunContext(self) as ctx:
+            cli.run(('-f', 'json', path))
+        print (ctx.stdout)
+        self.assertTrue(json.loads(ctx.stdout))
```

Comment:
I don't understand what you test here.

Again, can you please make a real check? For example:
```python
        with RunContext(self) as ctx:
            cli.run(('-f', 'json', path))
        expected_out = (
            '[{"path":"%s",...' % path +
            '"line":1,"char":1,...},'
            '{"path":"file2",...'
            '"line":1,"char":1,...},]\n')
        self.assertEqual(
            (ctx.returncode, ctx.stdout, ctx.stderr), (0, expected_out, ''))
```